src/components/projects/AddProject.vue

```diff
@@ -74,8 +74,8 @@ const handleDeleteProject = async (projectName) => {
       elevation="1"
     >
       <h3 class="ml-4">{{ capitalizeLetters(project) }}</h3>
-      <PrimaryButton variant="plain" icon>
-        <DeleteButton @click="() => handleDeleteProject(project)" />
+      <PrimaryButton variant="outlined" color="error" icon>
```

Comment:
Just a personal apreciation; I don't really like the outlined buttons. In this case, the button look so big in comparison with  the v-sheet. In mi opinion y you change the variant "plain" to "text" looks much better. What do you think?

Reply:
variant changed to "text" :heavy_check_mark: